### PR TITLE
[IMP] mail: introduce ChannelMemberListCategoryComponent

### PR DESCRIPTION
**Pull request by alexkuhn**
Task-2831082

### CODE REVIEW

File: addons/mail/static/src/components/channel_member_list_category/channel_member_list_category.xml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+<templates xml:space="preserve">
+
+    <t t-name="mail.ChannelMemberListCategory" owl="1">
+        <h6 class="m-2"><t t-esc="title"/> - <t t-esc="members.length"/></h6>
+        <t t-foreach="members" t-as="member" t-key="member.localId">
+            <ChannelMember channelLocalId="channelMemberListView.channel.localId" memberLocalId="member.localId"/>
+        </t>
+    </t>
+
+</templates>
```
Comment (oomsveta) [inline addons/mail/static/src/components/channel_member_list_category/channel_member_list_category.xml:11]: missing new line

File: addons/mail/static/src/components/channel_member_list_category/channel_member_list_category.xml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-name="mail.ChannelMemberListCategory" owl="1">
+        <h6 class="m-2"><t t-esc="title"/> - <t t-esc="members.length"/></h6>
```
Comment (oomsveta) [inline addons/mail/static/src/components/channel_member_list_category/channel_member_list_category.xml:5]: maybe merge these two properties and make it translatable

File: addons/mail/static/src/components/channel_member_list_category/channel_member_list_category.xml (hunk 1 above)
Comment (alexkuhn) [reply addons/mail/static/src/components/channel_member_list_category/channel_member_list_category.xml]: Change done

File: addons/mail/static/src/components/channel_member_list_category/channel_member_list_category.xml (hunk 2 above)
Comment (alexkuhn) [reply addons/mail/static/src/components/channel_member_list_category/channel_member_list_category.xml]: Change done

**Review by oomsveta — CHANGES_REQUESTED** [review]

File: addons/mail/static/src/components/channel_member_list_category/channel_member_list_category.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    get title() {
+        let categoryText = "";
+        if (this.props.category === 'online') {
+            categoryText = this.env._t("Online");
+        }
+        if (this.props.category === 'offline') {
+            categoryText = this.env._t("Offline");
+        }
+        return sprintf(
+            this.env._t("%s - $s"),
```
[the remark this answers is not in the document]
Comment (alexkuhn) [reply addons/mail/static/src/components/channel_member_list_category/channel_member_list_category.js]: Change done

[the remark this answers is not in the document]
Comment (alexkuhn) [reply addons/mail/static/src/components/channel_member_list_category/channel_member_list_category.js]: Change done

**Review by oomsveta — APPROVED** [review]

**Comment by oomsveta** [comment]
Need rebase. Otherwise looks good to me 🙂

**Comment by alexkuhn** [comment]
Merged in grouped PR: https://github.com/odoo/odoo/pull/91000

**Pull request closed without merging**